### ISSUE TITLE
Error running PHTN-67-000037 in Ansible Photon Playbook

### ISSUE THREAD

**Issue by scjohnson1988** [vmware/dod-compliance-and-automation#23]
The play under "PHTN-10-000037 - Set kernel parameter net.ipv4.tcp_syncookies" is failing to run on both VCSAs.  Error messages generated were on an airgapped system, but it related to failing to reload systemctl.  The error message ended as "\1nsysct: setting key \:fs.suid_Dumpable\":Invalid argument\n:}

We don't have much support for IPv6 routing, so there may be some correlation there.

**Comment by rlakey** [comment]
Hi @scjohnson1988 

What version of VCSA are you running it on?

If you just run a "sysctl -p" on the appliance does it report the error?

**Comment by scjohnson1988** [comment]
@rlakey I'll have to get back to you on this one.  I don't have access to that device unless I'm on site. I'll try to re-run that specific t ag and follow it up with sysctl.

For version info, it is on 6.7.0U3m (latest 6.7 VC release).

**Comment by scjohnson1988** [comment]
@rlakey I had a chance to test this.  Interestingly enough, when I ran the playbook with a tag of "PHTN-10-000037", it was fine.  Although, it didn't make any changes.  I ran a sysctl -p on the VCSA and it has one line of: `setting key "fs.suid_dumpable": Invalid argument`

That error is basically the same that I reported initially in the first post, which is what I saw when I did my first run of the whole playbook.
